spec/presenters/tree_builder_default_filters_spec.rb

```diff
@@ -4,62 +4,16 @@
-                                       :description => "Environment / Prod",
-                                       :options     => nil,
-                                       :db          => "MiqTemplate",
-                                       :search_type => "default",
-                                       :search_key  => "_hidden_"))
-      @filters.push(FactoryGirl.create(:miq_search,
-                                       :name        => "default_Environment / Prod",
-                                       :description => "Environment / Prod",
-                                       :options     => nil,
-                                       :db          => "Container",
-                                       :search_type => "default",
-                                       :search_key  => "_hidden_"))
-      @filters.push(FactoryGirl.create(:miq_search,
-                                       :name        => "default_Environment / Prod",
-                                       :description => "Environment / Prod",
-                                       :options     => nil,
-                                       :db          => "ContainerGroup",
-                                       :search_type => "default",
-                                       :search_key  => "_hidden_"))
-      @filters.push(FactoryGirl.create(:miq_search,
-                                       :name        => "default_Environment / Prod",
-                                       :description => "Environment / Prod",
-                                       :options     => nil,
-                                       :db          => "ContainerService",
-                                       :search_type => "default",
-                                       :search_key  => "_hidden_"))
-      @filters.push(FactoryGirl.create(:miq_search,
-                                       :name        => "default_Environment / Prod",
-                                       :description => "Environment / Prod",
-                                       :options     => nil,
-                                       :db          => "Storage",
-                                       :search_type => "default",
-                                       :search_key  => "_hidden_"))
-      @filters.push(FactoryGirl.create(:miq_search,
-                                       :name        => "default_Environment / Prod",
-                                       :description => "Environment / Prod",
-                                       :options     => nil,
-                                       :db          => "Vm",
-                                       :search_type => "default",
-                                       :search_key  => "_hidden_"))
+      @filters = [
+        MiqSearch.find_by(:name => "default_Platform / HyperV", :description => "Platform / HyperV", :options => nil, :db => "Host", :search_type => "default", :search_key => nil),
```

Comment:
Should these be `MiqSearch.find_by(:name => "sdfsd").create_with(:description => "sdfsdf", ...)` ?